gax-httpjson/src/main/java/com/google/api/gax/httpjson/ProtoMessageRequestFormatter.java

```diff
@@ -104,6 +117,7 @@ public PathTemplate getPathTemplate() {
     private FieldsExtractor<RequestT, Map<String, List<String>>> queryParamsExtractor;
     private String rawPath;
     private FieldsExtractor<RequestT, Map<String, String>> pathVarsExtractor;
+    private List<String> rawAdditionalPaths = Collections.emptyList();
```

Comment:
I think usually we set default values when instantiating builders, see this [doc](go/autovalue/builders-howto#default). I know this is a custom builder, but can we keep the same practice?

Reply:
Yes, good point. Changed to be more consistent with the rest of the code.